dss/events/handlers/index.py

```diff
@@ -4,9 +4,12 @@
 import os
 import re
 import uuid
-from urllib.parse import unquote
+import ipaddress
+import socket
+from urllib.parse import urlparse, unquote
 
 import requests
+from requests_http_signature import HTTPSignatureAuth
```

Comment:
@Bento007 observed that `requests_http_signature` is a recently developed package related to this PR. Should it also be reviewed? @Bento007 has an interest in security and could be a good candidate to do the review. This could also help develop broader knowledge within the team regarding our security mechanisms.

Reply:
Sure, review as much as you like.